src/sinks/datadog/metrics/service.rs

```diff
@@ -183,12 +194,13 @@ impl Service<DatadogMetricsRequest> for DatadogMetricsService {
                 .await
                 .context(CallRequestSnafu)?;
             let body = body.copy_to_bytes(body.remaining());
-
             Ok(DatadogMetricsResponse {
                 status_code: parts.status,
                 body,
                 batch_size,
                 byte_size,
+                raw_byte_size: byte_size,
```

Comment:
Just to clarify, this is the bytes _before_ compression, right?

Reply:
It's the compressed size, since we inherently compress as part of the incremental encoding process.

Reply:
Gotcha. I opened https://github.com/vectordotdev/vector/pull/12501 to make it more explicit in the component spec, but I think the metric should reflect _uncompressed_ serialized bytes to mirror `compenent_received_bytes_total`.